mobile/apps/artist/routes.coffee

```diff
@@ -26,6 +34,11 @@ module.exports.index = (req, res, next) ->
     artist.maybeFetchAndSetFeaturedBio()
     artist.fetchArtworks(data: {size: 50})
   ]).then ([artistData, partnerArtists, artworks]) ->
+    res.locals.sd.modal =
```

Comment:
The intention behind this is that `sd` is meant to hold vanilla data "constants", and often environment variables, that are uppercase.